tool/dart_site/lib/dart_site.dart

```diff
@@ -29,5 +31,7 @@ final class DartSiteCommandRunner extends CommandRunner<int> {
     addCommand(AnalyzeDartCommand());
```

Comment:
Do we have a "man dart_site" type of doc anywhere that documents these commands? That would be super helpful!

Reply:
You can run `dart run dart_site --help` to get a list of each command with their documentation. Once I've simplified their usage I'll be including how to use these in task-based site infra documentation.